small adjustments except and intersect docs (#652)
SimpleDataLabsInc/prophecy-docs: docs/analysts/development/gems/join-split/except.md

</div>

The output indicates that order `101` and `102` appear in **Table A**, but not in **Table B**.

#### Preserve duplicates

> If the `preserve duplicates` option is selected, the duplicates from Table A are preserved.

Reviewer: ```suggestion
If you select the `preserve duplicates` option, the gem preserves the duplicates from Table A.
```
Nit/Suggestion: Remove passive voice.

Also applies to the intersect page.

<div class="table-example">

| order_id | customer_id | order_date | amount |
| -------- | ----------- | ---------- | ------ |
| 102      | 2           | 2024-12-03 | 150.00 |
| 101      | 1           | 2024-12-01 | 250.00 |
| 101      | 1           | 2024-12-01 | 250.00 |

> The output is the same as above, except the duplicate order `101` is retained.

Reviewer: ```suggestion
The output is the same as above, except the gem also retains the duplicate order `101`.
```

Also applies to the intersect page.
